### PR TITLE
хуефикация

### PR DESCRIPTION
**Pull request by swined**

### CODE REVIEW

**Comment by swined** [comment]
пинг-хуинг

**Comment by dimitriyremerov** [comment]
What about https://github.com/dimitriyremerov/life/commit/51d892c53d483161d4e1612b034b7b1fe134a553?

**Comment by swined** [comment]
fixed

**Comment by dimitriyremerov** [comment]
I need a review and approval of at least one more person from the team.

File: manifest.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-* Не следить за здоровьем — плохо. Помни об этом. А дальше уж знаешь что делать.
+* Бегать 4 раза в неделю — это заебись. Иногда и реже норм, если уж обстоятельства такие. Иногда даже и совсем не бегать норм, если болеешь, например. Думай головой.
+* В качалку — тоже заебись. Ну смотри сам, как с бега время будет оставаться.
+* Пить в одного — хуёво. Хотя изредка и это норм.
+* Пить вдвоём часто — хуёво. Но допустимы исключения — иногда это норм.
+* Воображаемые друзья это заебись. Но пить с ними - хуёво.
+* Пить в большой компании это, в принципе, норм, особенно если почти всегда в нескольких определённых. А вот пить постоянно в разных больших компаниях — хуёво. Это как искать повод напиться, ища каждый день разные митапы только для отмазки.
+* Курить — хуёво. Даже иногда. Может совсем изредка это и норм, но решай сам.
+* Курить траву - заебись. Ты же в Амстердаме, там лигалайз. А лигалайз это заебись.
```
Comment (dimitriyremerov) [inline manifest.md]: Your dashes are different from long dashes in the original, this makes it inconsistent.

File: manifest.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+* Обжираться — хуёво. Целая курица на обед — это дохуя. Половина — норм. Целая порция индийской еды из Tandoori Express  — это дохуя. Половина — норм. Иногда можно и целую, но редко. 
+* Обжираться по накурке - заебись.
+* Пить часто колу из бесплатного холодильника в офисе — хуёво. Иногда — норм. Но только, если разбавляешь вискарём.
+* Писать на PHP - хуёво. Но если хорошо заплатят и потом не забыть вымыть руки, то норм.
```
Comment (dimitriyremerov) [inline manifest.md]: > Писать на PHP - хуёво. Но если хорошо заплатят и потом не забыть вымыть руки, то норм.

I don't agree on this one.

File: manifest.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+* Бегать 4 раза в неделю — это заебись. Иногда и реже норм, если уж обстоятельства такие. Иногда даже и совсем не бегать норм, если болеешь, например. Думай головой.
+* В качалку — тоже заебись. Ну смотри сам, как с бега время будет оставаться.
+* Пить в одного — хуёво. Хотя изредка и это норм.
+* Пить вдвоём часто — хуёво. Но допустимы исключения — иногда это норм.
+* Воображаемые друзья это заебись. Но пить с ними - хуёво.
+* Пить в большой компании это, в принципе, норм, особенно если почти всегда в нескольких определённых. А вот пить постоянно в разных больших компаниях — хуёво. Это как искать повод напиться, ища каждый день разные митапы только для отмазки.
+* Курить — хуёво. Даже иногда. Может совсем изредка это и норм, но решай сам.
+* Курить траву - заебись. Ты же в Амстердаме, там лигалайз. А лигалайз это заебись.
+* Обжираться — хуёво. Целая курица на обед — это дохуя. Половина — норм. Целая порция индийской еды из Tandoori Express  — это дохуя. Половина — норм. Иногда можно и целую, но редко. 
+* Обжираться по накурке - заебись.
```
Comment (dimitriyremerov) [inline manifest.md]: > Обжираться по накурке - заебись.

I don't necessarily agree on this one. Maybe we should use 'норм' instead?

**Comment by rootree** [comment]
Где кнопка смержить

**Comment by swined** [comment]
мерж пуллреквест, блеать!

**Comment by dimitriyremerov** [comment]
My comment on PHP statement was not resolver, we need to remove this one completely from the branch.

**Pull request merged**